Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1,256 +1,876 @@
-﻿<ReplaceList>
-  <WholeWords>
-    <Word from="čovek" to="čovjek" />
-    <Word from="podnjeti" to="podnijeti" />
-    <Word from="podneo" to="podnio" />
-    <Word from="ponjeti" to="ponijeti" />
-    <Word from="odnjeti" to="odonijeti" />
-    <Word from="donjeti" to="donijeti" />
-    <Word from="porodica" to="obitelj" />
-    <Word from="porodični" to="obiteljski" />
-    <Word from="savet" to="savjet" />
-    <Word from="pesma" to="pjesma" />
-    <Word from="mesec" to="mjesec" />
-    <Word from="mestu" to="mjestu" />
-    <Word from="mesto" to="mjesto" />
-    <Word from="videti" to="vidjeti" />
-    <Word from="poonaosob" to="osobno" />
-    <Word from="veče" to="večer" />
+<ReplaceList>
+<WholeWords>
+	<Word from="aluminijum" to="aluminij" />
+	<Word from="abstraktno" to="apstraktno" />
+	<Word from="abstraktna" to="apstraktna" />
+	<Word from="bedni" to="bijedni" />
+	<Word from="bednog" to="bijednog" />
+	<Word from="beleg" to="biljeg" />
+	<Word from="belešci" to="bilješci" />
+	<Word from="bezbjedna" to="sigurna" />
+	<Word from="bioskop" to="kino" />
+	<Word from="bitci" to="bitki" />
+	<Word from="bekstvo" to="bijeg" />
+	<Word from="beše" to="bješe" />
+	<Word from="braon" to="smeđa" />
+	<Word from="bukvalno" to="doslovno" />
+	<Word from="ceo" to="cijeli" />
+	<Word from="cjeli" to="cijeli" />
+	<Word from="celog" to="cijelog" />
+	<Word from="cjelog" to="cijelog" />
+	<Word from="celu" to="cijelu" />
+	<Word from="čas " to="sat " />
+	<Word from="čemo" to="ćemo" />
+	<Word from="ču" to="ću" />
+	<Word from="češ" to="ćeš" />
+	<Word from="če" to="će" />
+	<Word from="ćošku" to="uglu" />
+    <Word from="ćerka" to="kći" />
+    <Word from="kćerka" to="kći" />
+    <Word from="ćutao" to="šutio" />
+    <Word from="ćutala" to="šutjela" />
+    <Word from="daga" to="da ga" />
+    <Word from="dejstvo" to="djelovanje" />
+	<Word from="dela" to="djela" />
+	<Word from="delo" to="djelo" />
+	<Word from="daćeš" to="dat ćeš" />
+ 	<Word from="daće" to="dat će" />
+	<Word from="delovala" to="djelovala" />
+	<Word from="delovanje" to="djelovanje" />
+	<Word from="delu" to="dijelu" />
+	<Word from="deo" to="dio" />
+	<Word from="dešava" to="događa" />
+	<Word from="dete" to="dijete" />
+	<Word from="detetu" to="djetetu" />
+	<Word from="deteta" to="djeteta" />
+	<Word from="dječiji" to="dječji" />
+	<Word from="dijetetom" to="djetetom" />
+	<Word from="dobo" to="dobro" />
+	<Word from="doćiću" to="doći ću" />
+	<Word from="dole" to="dolje" />
+	<Word from="donjeti" to="donijeti" />
+	<Word from="dospeo" to="dospio" /> /*nije za regex*/
```

Comment:
I think, the XML comment syntax would be `<!-- nije za regex -->` instead of `/*nije za regex*/`.
As in [this hrv_OCRFixReplaceList.xml](https://raw.githubusercontent.com/xylographe/subtitleedit/hrvofr/Dictionaries/hrv_OCRFixReplaceList.xml).